tox.ini

```
@@ -0,0 +1,11 @@
+[tox]
+envlist =
+    py37,
+    py37-flake8
+
+[testenv]
+commands = python ./runtests.py
+
+[testenv:flake8]
+deps = py37-flake8
```

Think this should just be flake8
```suggestion
deps = flake8
```